doc/build-unix.md

```diff
@@ -108,27 +101,58 @@ According to installed dependencies, the following steps will compile `dogecoind
 make
 make install # optional
 ```
-See [Build configuration](#build-configuration) section for extra settings.
+See [Build configuration](#build-configuration) for extra settings.
 
 ## Build configuration
-You can see all available options with `./configure --help`
+
+Configurations are done during the `./configure` step. Use `--help` to see all available options.
 
 #### Enable Qt GUI
 Create `dogecoin-qt`, the core wallet GUI.
-```
+```bash
 ./configure --with-gui
 ```
 
-#### Incompatible BerkeleyDB
-If you're using any other BerkeleyDB version than 4.8 :
+#### Wallet
+BerkeleyDB is required for wallet functionality and use the `wallet.dat` file.
+
+By default, **Dogecoin Core expect BerkeleyDB 5.1**.  
+You can use a different version by specifying `--with-incompatible-bdb` flag.
+
+If no package is available for your distribution  in optional dependencies, you can build BerkeleyDB from source :
+```bash
+# Install script for BerkeleyDB 5.1
+
+# BerkeleyDB installation directory
+BDB_PREFIX=$(pwd)/bdb
+mkdir $BDB_PREFIX
+
+# Fetch the source and verify shasum
+wget 'http://download.oracle.com/berkeley-db/db-5.1.29.NC.tar.gz'
+echo '08238e59736d1aacdd47cfb8e68684c695516c37f4fbe1b8267dde58dc3a576c db-5.1.29.NC.tar.gz' | sha256sum -c
```

Comment:
this is working
![image](https://user-images.githubusercontent.com/3387385/123412003-61b34b80-d5b1-11eb-899c-a497b3dae28a.png)